src/types/hbs.rs

```diff
@@ -6,15 +6,43 @@ use hpos_hc_connect::hha_agent::HHAAgent;
 use hpos_hc_connect::hpos_agent::get_hpos_config;
 use reqwest::Response;
 use serde::{Deserialize, Serialize};
+use tracing::warn;
 
-#[derive(Debug, Deserialize)]
-pub struct HostingCriteria {
+#[derive(Debug, Deserialize, Clone)]
+pub struct HostCredentials {
     #[allow(dead_code)]
     pub id: Option<String>,
     pub jurisdiction: Option<String>,
-    pub kyc: KycLevel,
+    pub kyc: Option<KycLevel>,
```

Comment:
It is an option on the returned TS type in hbs.  Therefore, I think it's most accurate to show it as an option when deserializing it here.

Reply:
Yes, but given everything is an option when you deserialize it will always succeed even if no value is returned. This would be a waste of a check. And since the only value you need is the KYC it would be good to expect a response.

Reply:
Ok. Fair point, I'll update it.